workspaces/ui-v2/src/pages/docs/EndpointRootPage.tsx

```diff
@@ -218,10 +230,11 @@ export const EndpointRootPage: FC<
           }
         />
 
-        {bodies.requests.map((i, index) => {
+        {bodies.requests.map((i) => {
           return (
             <TwoColumnBodyEditable
-              key={index}
+              key={i.rootShapeId}
```

Comment:
Yes, nice one! Using array index as component keys is a mistake that leads to some nasty rendering bugs / incorrect react component instancing!